env/file_system_tracer.cc

```diff
@@ -338,6 +338,24 @@ IOStatus FSRandomAccessFileTracingWrapper::InvalidateCache(size_t offset,
   return s;
 }
 
+IOStatus FSRandomAccessFileTracingWrapper::ReadAsync(
```

Comment:
@mrambacher had a good point about this tracing only the initiation of the read. To get the full trace, this needs to intercept the callback, and possibly allocate its own cb_arg to save the len/offset and other info. I think tracing can be deferred to a subsequent PR when these APIs are actually used.